external/fv3fit/fv3fit/keras/_training.py

```diff
@@ -19,7 +23,7 @@ def get_model(
     sample_dim_name: str,
```

Comment:
Haven't looked through the tests enough to see why this function is still needed, but it would be nice to be able to get rid of it since it's no longer used by the training routine. OTOH this PR is pretty big already...